cypress/integration/rancid_tomatillos_spec1.js

```diff
@@ -22,19 +22,19 @@ describe('App', () => {
         statusCode: 422,
       })
       cy.visit('http://localhost:3000')
-      .get('h2').should('contain', 'Oh wow. This is embarassing. Try again later? 😅')
+      .get('h2').contains('This is a 400 error message on the Movie Library Page')
```

Comment:
Interesting change here; do you prefer this syntax? I wonder if there is any sort of benefit of using one over the other or if it's just a matter of preference